python/ray/rllib/examples/multiagent_mountaincar.py

```diff
@@ -48,7 +48,7 @@ def create_env(env_config):
     config["model"].update({"fcnet_hiddens": [256, 256]})
     options = {"multiagent_obs_shapes": [2, 2],
                "multiagent_act_shapes": [1, 1],
-               "multiagent_shared_model": False,
+               "is_shared_model": True,
```

Comment:
should we keep the multiagent_ prefix?

Reply:
Sure, would prefixing it with the 'is' be good since it is a boolean?

Reply:
yeah, that makes sense